web/src/components/Plugin/UI/authz-casbin.tsx

```diff
@@ -37,54 +38,122 @@ const FORM_ITEM_LAYOUT = {
 const AuthzCasbin: React.FC<Props> = ({ form, schema }) => {
   const { formatMessage } = useIntl();
   const properties = schema?.properties
+  const [value, setValue] = useState(1);
+  const onChange = (e:any) => {
+    setValue(e.target.value);
+  };
+
+  const [dian, setDian] = useState('houmian')
+  const aa = () => {
+    setDian('houmian')
+  }
+
+  const bb = () => {
+    setDian('qianmian')
+  }
```

Comment:
Hi, these variable names, are not particularly friendly, we can change the variable names